### ISSUE TITLE
Don't use absolute path as install destination in CMakeLists.txt

### ISSUE THREAD

**Issue by db-tech** [ultravideo/uvgRTP#13]
I would recommend not using absolute paths as install destination like those:

```      
 install(TARGETS uvgrtp
                ARCHIVE
                DESTINATION /usr/local/lib
        )

    install(DIRECTORY include/ DESTINATION /usr/local/include/uvgrtp
            FILES_MATCHING PATTERN "*.hh"
    )

```

Maye you could use something like this ?:
```
# Define install target, install libraries and archives (static libraries) to "<prefix>/lib"
install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}Targets
        LIBRARY DESTINATION lib
        ARCHIVE DESTINATION lib
        INCLUDES DESTINATION include)

#Copy all header files to the <prefix>/include/<projectName> directory with subdirectories retained
install(DIRECTORY ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}
        DESTINATION include
        FILES_MATCHING PATTERN "*.hh")
```

**Comment by altonen** [comment]
Hi,

please open a pull request, I can test it at some point and give it LGMT.

**Comment by db-tech** [comment]
K, will do that when I have the time.

**Comment by polusto** [comment]
Merged with #44
